src/calendar/calendar.js

```diff
@@ -1,4 +1,7 @@
 import { Calendar } from "./calendar-class.js";
+//import { populateDefaultLog } from "../dailyLog/dailyLog.js";
```

Comment:
why are these commented out?

Reply:
That was from an old version of the code. Has since been removed and updated after pulling from dev once PR#106 was merged.